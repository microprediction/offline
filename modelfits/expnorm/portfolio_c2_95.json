{"g1": 0.02731770888350597, "g2": 7.280704621946694, "logK": 0.26513164451612947, "loc": 0.047862674683636225, "logScale": 0.1050921001729167}