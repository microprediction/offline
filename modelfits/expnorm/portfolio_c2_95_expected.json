{"g1": 0.38637461975473253, "g2": 12.832609372755563, "logK": 0.9936721583812147, "loc": -0.04243312099705092, "logScale": -2.6043387648978644}